common/changes/office-ui-fabric-react/master_2018-07-09-18-03.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "added fourth paragraph to ListOverview.md that explains how to format the items array if they opt to not use the optional onRenderCell",
+      "type": "patch"
```

Comment:
You could even edit this to `none` if you'd like since this is a documentation-only update